src/services/tx/txSender.ts

```diff
@@ -71,7 +71,7 @@ export const createTx = async (txParams: SafeTransactionDataPartial, nonce?: num
 export const createMultiSendTx = async (txParams: MetaTransactionData[]): Promise<SafeTransaction> => {
```

Comment:
I mean, you don't need this function now, it doesn't add anything on top of createTx.